services/storage/object/controls.md

```diff
@@ -1,11 +1,139 @@
-# Object Storage Controls
+# CCC.OS: Object Storage v25.07
```

Comment:
```suggestion
# CCC.OS: Object Storage
```

Per suggestion from the Delivery WG, we likely don't want the version number in the document itself, since it risks being unaligned with the corresponding release

Reply:
Fixed for @mlysaght2017 while fixing minor formatting issue.